versioned_docs/version-v2.16.x/user-guide/systemrequirements-zos.md

```diff
@@ -1,5 +1,5 @@
 ---
-keywords: [security permissions, system permissions, monacat]
```

Comment:
This change was from copy-pasting the v2.18 page.
Not sure what monacat is, seems to have been removed in newer versions of the doc.